var/spack/repos/builtin/packages/r-gdalutils/package.py

```diff
@@ -21,3 +21,4 @@ class RGdalutils(RPackage):
     depends_on('r-utils', type=('build', 'run'))
```

Comment:
This is something that I have on my TODO list that I was going to bring up after the dust settled in this PR. Now is a good time though. There is no `r-r-utils` package in Spack. The `r-utils` package is actually for `R.utils`. This is confusing because there is a base package named `utils` and I saw some cases where that was used when the spec was actually for the base package. The bottom line is that the `r-utils` package should be called `r-r-utils` to be consistent with other package naming. The same is true for the `r-oo` package, it should be called `r-r-oo`, as it corresponds to R.oo. At least that one does not have the same name as a base package.

Reply:
Sounds fine by me, I think it's few enough packages that I don't really care. I'll let you decide.

Reply:
Is there a process for renaming Spack packages, other than the git operations?